README.md

```diff
@@ -4,117 +4,6 @@ This is a command line interface for Pulp 3.
 
 This software is in beta and future releases may include backwards incompatible changes.
 
-## General command syntax
+Check out our [docs](docs/) and the [quickstart guide](docs/quickstart.md) to get started.
 
-`pulp [<global_options>] <plugin> <resource_class> [--type <resource_type>] <action> [<action_options>]`
 
-Global options include:
-
-  * `--base-url`
-  * `--user`
-  * `--password`
-  * `--config`
-  * `--format`
-  * `-v`/`-vv`/`-vvv`
-
-## Quick start
-
-To install and use the CLI, run these commands:
-
-```
-pip install pulp-cli[pygments]
-pulp config create -e
-```
-
-Read [the installation and configuration doc](docs/install.md) for more information.
-
-## Known issues
-
-  * Redirecting from `http` to `https`, as done by a typical Pulp installation,
-    does not work properly with `POST` and `PUT` requests.
-    Please use `https://` in the base url.
-    Note that the attempt to use `http` leaks sensitive data over an unencrypted connection.
```

Comment:
FYI, I dropped this. I don't think it's an issue anymore since we don't follow redirects.

Reply:
So what happens now, if you try to use the wrong schema?

Reply:
You get a connection error.

Reply:
^ That's if there's no redirect.

If there's a redirect, then users receive [this error](https://github.com/pulp/pulp-cli/blob/42f62321e61465b28c9dcbc0c9822538631dcb16/pulpcore/cli/common/openapi.py#L290-L294)